packages/protocol/docs/how_taiko_proves_blocks.md

```diff
@@ -106,7 +106,7 @@ struct BlockMetadata {
 
 ### Global Variables
 
-The following [**block level variables**](https://docs.soliditylang.org/en/v0.8.18/units-and-global-variables.html) are accessible to the EVM, but their values are not part of the MPT so we need a different way to verify their correctness.
+The following [**block level variables**](https://docs.soliditylang.org/en/v0.8.25/units-and-global-variables.html) are accessible to the EVM, but their values are not part of the MPT so we need a different way to verify their correctness.
```

Comment:
change to use `https://docs.soliditylang.org/en/latest/units-and-global-variables.html` please.